docs/lang/booleans.md

```diff
@@ -0,0 +1,328 @@
+# Booleans
+
+[[Back to all operators]](./standard-operators.md)
+
+You find these operators in every programming language and every textbook on
+logic. These operators form _propositional logic_.
+
+<a name="const"></a>
+## Constants
+
+TLA+ contains three special constants: `TRUE`, `FALSE`, and `BOOLEAN`.
+The constant `BOOLEAN` is defined as the set `{FALSE, TRUE}`.
+
+In theory, `TRUE` and `FALSE` are also sets, but in practice they are treated
+as indivisible values. For instance, Apalache and TLC will report an error, if
+you try to treat `FALSE` and `TRUE` as sets or integers.
+
+## Operators
+
+**Warning**: Below, we discuss Boolean operators in a way as they are usually
+defined in programming languages. It is important to understand the effects `F
+\/ G`, when `F` and `G` contain the operator prime (`'`), or when they are used
+inside the initialization predicate `Init`. We discuss this in [Control Flow
+and Non-determinism].
+
+----------------------------------------------------------------------------
+
+<a name="and"></a>
+### And (conjunction)
+
+**Notation:** `F /\ G` or `F \land G`
+
+**LaTeX notation:** ![land](./img/land.png)
+
+**Arguments:** Two or more arbitrary expressions.
```

Comment:
Well, because TLA+ does not enforce types. You can pass arbitrary expressions. It's up to semantics and a tool how to evaluate it. To make things worse, there are three kinds of TLA+ semantics, when types are concerned.